src/main/java/Entities/Ingredient.java

```diff
@@ -1,8 +1,10 @@
 package main.java.Entities;
 
+import main.java.EntityInterfaces.IngredientImpl;
+
 import java.util.List;
 
-public class Ingredient implements main.java.EntityInterfaces.Ingredient {
+public class Ingredient implements IngredientImpl {
     private String name;
```

Comment:
Sorry if I confused you, the interface should be `XYZ` and the class should be `XYZImpl`. You did that backwards. The "impl" stands for "implementation".